odoo/openupgrade/openupgrade_loading.py

```diff
@@ -157,6 +157,8 @@ def isrelated(model, k):
                     sorted([x[0] for x in v.selection]))
             else:
                 properties['selection_keys'] = 'function'
+        elif v.type == 'binary':
+            properties['attachment'] = str(getattr(v, "attachment", False))
```

Comment:
Ah, I see that compare.py is not shared in openupgradelib, but instead kept locally in odoo/addons/openupgrade_records/lib/. So we can include this here and then add the 'attachment' key in Odoo 11 to make it show up in the analysis files.

compare.py does however assume that keys are present for every field, not just binary fields. So you should add a default in the properties dictionary right above.